<tt>
Hi&nbsp;guys,&lt;br&gt;&lt;br&gt;me&nbsp;again,&nbsp;found&nbsp;another&nbsp;Moq&nbsp;related&nbsp;VM&nbsp;bug.&nbsp;Submitted&nbsp;here:&nbsp;&lt;a&nbsp;href=&quot;https://bugzilla.novell.com/show_bug.cgi?id=473482&quot;&gt;https://bugzilla.novell.com/show_bug.cgi?id=473482&lt;/a&gt;&lt;br&gt;&lt;br&gt;This&nbsp;one&nbsp;is&nbsp;peculiar,&nbsp;it&nbsp;fails&nbsp;on&nbsp;line:&nbsp;var&nbsp;t&nbsp;=&nbsp;ex.StackTrace;&lt;br&gt;<br>
&lt;br&gt;Regards,&nbsp;Bas&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;&lt;br&gt;using&nbsp;System;&lt;br&gt;&lt;br&gt;public&nbsp;class&nbsp;Program&lt;br&gt;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;static&nbsp;void&nbsp;Main()&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;try&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;new&nbsp;Foo&amp;lt;object&amp;gt;(0,&nbsp;0);&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;}&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;catch&nbsp;(Exception&nbsp;ex)&lt;br&gt;<br>
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;var&nbsp;t&nbsp;=&nbsp;ex.StackTrace;&nbsp;&amp;lt;==&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;}&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;}&lt;br&gt;}&lt;br&gt;&lt;br&gt;public&nbsp;class&nbsp;Foo&amp;lt;T&amp;gt;&nbsp;:&nbsp;FooBase&lt;br&gt;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;int&nbsp;i;&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;int&nbsp;ii;&lt;br&gt;&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;public&nbsp;Foo(int&nbsp;i,&nbsp;int&nbsp;ii)&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;this.i&nbsp;=&nbsp;i;&lt;br&gt;<br>
&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;this.ii&nbsp;=&nbsp;ii;&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;this.iii&nbsp;=&nbsp;0;&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;throw&nbsp;new&nbsp;Exception();&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;}&lt;br&gt;}&lt;br&gt;&lt;br&gt;public&nbsp;abstract&nbsp;class&nbsp;FooBase&lt;br&gt;{&lt;br&gt;&amp;nbsp;&amp;nbsp;&amp;nbsp;&nbsp;protected&nbsp;int&nbsp;iii;&lt;br&gt;}<br>

</tt>
